mqt/syrec/syrec_editor.py

```diff
@@ -161,11 +162,54 @@ def setup_actions(self):
         self.sim_action = QAction(QIcon.fromTheme('x-office-spreadsheet'), '&Sim...', self.parent)  # system-run
         self.stat_action = QAction(QIcon.fromTheme('applications-other'), '&Stats...', self.parent)
 
+        self.check1 = QCheckBox("Sythesizer with Add. Lines")
+        self.check1.setFont(QFont("Sanserif", 13))
+        self.check1.stateChanged.connect(self.item_selected)
+
+        self.check2 = QCheckBox("Sythesizer with no lines")
+        self.check2.setFont(QFont("Sanserif", 13))
+        self.check2.stateChanged.connect(self.item_selected)
```

Comment:
these should probably be called `check0` and `check1` to better align with the indices in the `buildType` array.
A cleaner solution would be to replace the `buildType` array with two boolean values, e.g., `synthesize_with_additional_lines` and `synthesize_without_additional_lines`.
Then the checkboxes can be called more appropriately as well.

Thinking a bit more about it:
this is not the appropriate GUI design for such an option. Please look into https://pythonpyqt.com/qbuttongroup/
This should be a group of radio buttons. One of them being selected per default.